src/Bookings.js

```diff
@@ -13,6 +15,7 @@ const Bookings = () => {
       <div className="container">
         <Search search={search} />
         {/* <SearchResults results={FakeBookings} /> */}
```

Comment:
You need to render the search results here - currently getting an error because you're not importing these components.